### PR TITLE
Pop window

### PR DESCRIPTION
**Pull request by Simpleshaikh1**
I created a pop window that displays nav menu upon clicking hanburger. 

Upon clicking the links it navigates to the respective pages.

### CODE REVIEW

**Review by amejid — COMMENTED** [review]
Hi @Simpleshaikh1, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has a failing linter check and does not have commits from your pair-programming partner. This is a collaborative activity.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by AbelG101 — COMMENTED** [review]
# Hi @Simpleshaikh1 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- You have submitted a project, that has a failing linters check, doesn't have the commits from your pair-programming partner and the commits you made are not verified. 
- Please make sure you make a commit on a verified Github account.
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html [hunk 1]
```diff
@@ -9,27 +9,32 @@
...
-                    <a href="#">Portfolio</a>
-                    <a href="#">About</a>
-                    <a href="#">Contact</a>
+                    <div class="x-men"><span>X</span></div>
+
+                    <a href="#portfolio" class="lis active">Portfolio</a>
+                    <a href="#about" class="lis">About</a>
+                    <a href="#contact" class="lis">Contact</a>
+
                 </nav>
```
Comment (mikethreels) [inline index.html:38]: - [ ] please make sure that you pop-up window looks similar to the one in de figma template provided by Microverse:
Your project:
afbeelding
Figma template:
afbeelding

File: index.html [hunk 2]
```diff
@@ -9,27 +9,32 @@
...
-                    <img src="./assets/Union.png" alt="">
+                    <img src="./assets/Union.png" alt="" >
     
                 </nav>
 
                 <nav class="nav-li">
-                    <a href="#">Portfolio</a>
-                    <a href="#">About</a>
-                    <a href="#">Contact</a>
+                    <div class="x-men"><span>X</span></div>
```
Comment (mikethreels) [inline index.html:32]: - [ ] when clicking on the "X" the menu pop-up doesn't close, it only closes when clicking on one of the links or just under the "X" please fix this.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Toyyib, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- great job making the links to the specific parts of the page work.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by binhussen — APPROVED** [review]
Hi @Simpleshaikh1 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**